Cargo.toml

```diff
@@ -10,6 +10,7 @@ license = "MIT / Apache-2.0"
 [features]
 default = []
 plugins = ["serde", "serde_macros", "heapsize", "heapsize_plugin"]
+rustc_stable = []
```

Comment:
Other servo dependencies with a cargo feature to make use of nightly rust has their's named "unstable" (e.g. [string-cache](https://github.com/servo/string-cache/blob/02f022def51ccf6c0d9383e8468246bf4f1b2242/Cargo.toml#L25), [tendril](https://github.com/servo/tendril/blob/fa4ecc3dc9f8677152b7d9139e636bd92f07d302/Cargo.toml#L21) and [heapsize](https://github.com/servo/heapsize/blob/787db63644eae687b60bee3a2ac48d8630da2351/Cargo.toml#L10)).
I therefore think "rustc_stable" should have it's meaning inverted and be renamed to "unstable"?